vm-virtio/src/queue.rs

```diff
@@ -424,6 +424,12 @@ pub struct Queue {
 
     #[serde(skip)]
     pub iommu_mapping_cb: Option<Arc<VirtioIommuRemapping>>,
+
+    /// VIRTIO_F_RING_EVENT_IDX negotiated
+    event_idx: bool,
+
+    /// The last used value when using EVENT_IDX
+    signalled_used: Option<Wrapping<u16>>,
```

Comment:
I was worried about snapshot/restore, but these new fields don't need to be restored as they are built when the device is created.

Reply:
Yup!